MVCTask/MVCTask/Controllers/MVCTaskActionInvoker.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Data.Entity;
+using System.Web.Mvc;
+
+namespace MVCTask.Controllers
+{
+    public class MVCTaskActionInvoker:ControllerActionInvoker
```

Comment:
It was made in IoC task just to learn how to register custom implementations of mvc classes. Transaction it's responsibility of data service.